lenses_python/SqlHandler.py

```diff
@@ -117,41 +67,33 @@ def _ConvertToDF(self, data):
         data = pd.DataFrame(data)
         return data
 
-    # def _ConvertToDF(self, data):
-    #     """
-    #     Get data from sql handler and extract from generate dict the messages and then the dict-value from each one
-    #     :param data: dictionary
-    #     :return: pandas dataframe
-    #     """
-    #     # Data has two keys messages/data and offsets
-    #     if data.get("messages", None) is not None:
-    #         data = data["messages"]
-    #         key = "messages"
-    #     elif data.get("data", None) is not None:
-    #         data = data["data"]
-    #         key = "data"
-    #     else:
-    #         raise Exception("There isn't key messages or data to exctract values for create pandas dataframe.\n")
-    #     # Get key value , which is a stringfy dict, loads convert it to dict
-    #     if key != "data":
-    #         data = list(map(lambda x: loads(x["value"]), data))
-    #     else:
-    #         data = list(map(lambda x: loads(x["value"]), data["messages"]))
-    #         temp_list = []
-    #
-    #     if len(self.datetimelist) > 0 and len(self.formatinglist) > 0:
-    #         # If these two lists has length greater than zero , then call class ConvertDateTime which
-    #         # which convert specific keys ,which have datetime string to datetime object
-    #         # this convert can be only if user request data as pandas dataframe
-    #         data = ConvertDateTime(data, self.datetimelist, self.formatinglist).Convert()
-    #     # Convert data to dataframe
-    #     data = pd.DataFrame(data)
-    #     return data
-
-
-
-
-
-
-
+    def browsing_data(self, extract_pandas=0):
+        """
 
+        :return:
+        """
+        params = {
+                   "token": self.token,
+                   "sql": self.query,
+                   "stats": 2
+                 }
+        params = urlencode(params)
+        url = self.url.replace("https", "wss")+"/api/sql/execute?"+params
```

Comment:
could you elaborate a little bit more? why you do a replace here? I mean you have a url in http and you change this url to `wss`? are you sure that this doesn't affect other functionalities when you have an instance of `SQLHandler` because you change a property of the class.

Reply:
Here I need to use wss://master...../api/sql... so, it's not need user to parse the new url for this method . I just replace the https with wss . the replace in self.url it's not affect the value of it. If I call in another method the it still keep the initial value. In any case I didnt use somewhere else